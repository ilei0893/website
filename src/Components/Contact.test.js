import React from "react";
import Contact from "./Contact.js";
import renderer from "react-test-renderer";

describe("Tests the contact section", () => {
  it("matches the snapshot", () => {
    const tree = renderer.create(<Contact />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
