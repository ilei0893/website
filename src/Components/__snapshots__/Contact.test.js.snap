// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests the header section matches the snapshot 1`] = `
<section
  id="contact"
>
  <div
    className="row section-head"
  >
    <div
      className="two columns header-col"
    >
      <h1>
        <span>
          Get In Touch.
        </span>
      </h1>
    </div>
    <div
      className="ten columns"
    >
      <p
        className="lead"
      />
    </div>
  </div>
  <div
    className="row"
  >
    <div
      className="eight columns"
    >
      <form
        action=""
        id="contactForm"
        method="post"
        name="contactForm"
      >
        <fieldset>
          <div>
            <label
              htmlFor="contactName"
            >
              Name 
              <span
                className="required"
              >
                *
              </span>
            </label>
            <input
              defaultValue=""
              id="contactName"
              name="contactName"
              size="35"
              type="text"
            />
          </div>
          <div>
            <label
              htmlFor="contactEmail"
            >
              Email 
              <span
                className="required"
              >
                *
              </span>
            </label>
            <input
              defaultValue=""
              id="contactEmail"
              name="contactEmail"
              size="35"
              type="text"
            />
          </div>
          <div>
            <label
              htmlFor="contactSubject"
            >
              Subject
            </label>
            <input
              defaultValue=""
              id="contactSubject"
              name="contactSubject"
              size="35"
              type="text"
            />
          </div>
          <div>
            <label
              htmlFor="contactMessage"
            >
              Message 
              <span
                className="required"
              >
                *
              </span>
            </label>
            <textarea
              cols="50"
              id="contactMessage"
              name="contactMessage"
              rows="15"
            />
          </div>
          <div>
            <button
              className="submit"
            >
              Submit
            </button>
            <span
              id="image-loader"
            >
              <img
                alt=""
                src="images/loader.gif"
              />
            </span>
          </div>
        </fieldset>
      </form>
      <div
        id="message-warning"
      >
         Error boy
      </div>
      <div
        id="message-success"
      >
        <i
          className="fa fa-check"
        />
        Your message was sent, thank you!
        <br />
      </div>
    </div>
    <aside
      className="four columns footer-widgets"
    >
      <div
        className="widget widget_contact"
      >
        <h4>
          Address and Phone
        </h4>
        <p
          className="address"
        >
          <br />
           
          <br />
          , 
           
          <br />
          <span />
        </p>
      </div>
    </aside>
  </div>
</section>
`;
